lib/components/tag-chip/index.tsx

```diff
@@ -1,24 +1,20 @@
-import React from 'react';
+import React, { FunctionComponent } from 'react';
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
-import { noop } from 'lodash';
 
-export const TagChip = ({ onSelect = noop, selected, tag: tagName }) => (
+type OwnProps = {
+  onSelect: () => {};
```

Comment:
it's probably helpful to follow suit with our other prop types by using `() => any` here